proposals/NNNN-Function-to-control-the-priority-of-audio-and-video-when-switching-between-apps.md

```diff
@@ -0,0 +1,137 @@
+| App1 STREAMABLE | STREAMABLE Status of App1  after App2 is launched | NOT_STREAMABLE=0;<br>STREAMABLE=1; |
+| App2 HMI_LEVEL | HMI Status of App2 after it is launched | BACKGROUND=0;<br>FULL=1;<br>LIMITED=2;<br>NONE=3; |
+| App2 AUDIBLE | AUDIBLE Status of App2 it is launched | NOT_AUDIBLE=0;<br>AUDIBLE=1; |
+| App2 STREAMABLE | STREAMABLE Status of App2 it is launched | NOT_STREAMABLE=0;<br>STREAMABLE=1; |
+
+Note:
+There are a total of 25 patterns of app combinations (App1 with 5 types x App2 with 5 types) by types.
+
+The SDL specifications for the status transition are below. If OEMs define customized status transition rule, it is necessary to follow the rules below.
+
+- Only one app can be FULL at the same time.
+- Only one app can be STREAMABLE at the same time.
+- In case two apps are same AppType, only one app can be AUDIBLE.
+
+Setting samples are below.
+
+<b>Table 2.</b> Example 1
+
+| Field | Value |
+| :-- | :-: |
+| App1 | NAVIGATION |
+| App2 | MEDIA |
+| App1 HMI_LEVEL | 0<br>(BACKGROUND) |
+| App1 AUDIBLE | 0<br>(NOT_AUDIBLE) |
+| App1 STREAMABLE | 0<br>(NOT_STREAMABLE) |
+| App2 HMI_LEVEL | 1<br>(FULL) |
+| App2 AUDIBLE | 1<br>(AUDIBLE) |
+| App2 STREAMABLE | 1<br>(STREAMABLE) |
+
+In this case, the status of App1 becomes BACKGROUND, NOT_AUDIBLE and NOT_STREAMABLE after the App2 is launched.
+At the same time, the status of App2 becomes FULL, AUDIBLE and STREAMABLE.
+Head unit (HU) screen display switched to App2 and audio is played from App2.
+
+<b>Table 3.</b> Example 2
+
+| Field | Value |
+| :-- | :-: |
+| App1 | PROJECTION<br>(isMedia = False) |
+| App2 | MEDIA |
+| App1 HMI_LEVEL | 2<br>(LIMITED) |
+| App1 AUDIBLE | 0<br>(NOT_AUDIBLE) |
+| App1 STREAMABLE | 1<br>(STREAMABLE) |
+| App2 HMI_LEVEL | 1<br>(FULL) |
+| App2 AUDIBLE | 1<br>(AUDIBLE) |
+| App2 STREAMABLE | 0<br>(NOT_STREAMABLE) |
+
+In this case, the status of App1 becomes LIMITED, NOT_AUDIBLE after the App2 is launched, but STREAMABLE status is kept as STREAMABLE.
+At the same time, the status of App2 becomes FULL, AUDIBLE and NOT_STREAMABLE.
+HU screen display switched to App2 and basic audio is ready to be played from App2. App1 is not displayed on the screen, but playback of the video continues in background.
+
+#### Error processing
+
+When the SDL is launched, the SDL Core loads two configration files, defaut configration and customized one by the OEM. Basically, the SDL Core uses the OEM's configuration file. But if it does not meet the status transiton specification as SDL, the SDL Core discards OEM's configuration file and uses setting in default configuration file. Table 4 is example of the invalid the OEM customized status transiton (in this transiton rule, 2 apps can be set STREAMABE status at the same time). In this case, the SDL Core discards only the non-conforming part of the setting provided by the OEM's configuration file and uses the value of default one.
```

Comment:
```suggestion
When the SDL is launched, SDL Core loads two configuration files, the default configuration and the customized one by the OEM. Basically, SDL Core uses the OEM's configuration file, but if it does not meet the status transition specification of the SDL app, SDL Core discards the OEM's configuration file and uses the setting in the default configuration file. Table 4 is example of the invalid errors in the OEM's customized status transition (in this transition rule, 2 apps can be set with STREAMABLE status at the same time). In this case, SDL Core discards only the non-conforming part of the setting provided by the OEM's configuration file and uses the value of default one.
```
Please also specify which SDL Component you're referencing in "When the SDL is launched," should this be "When the SDL app is launched,"?

Reply:
Hi, @theresalech -san.
Thank you for your review.
I understood. I will modify this sentence to "the status transition specification of the SDL Core".
Also, I will chage the sentence from "the status transition specification of the SDL Core" to "the status transition specification of the SDL Core".
Then, I will make a commit them.
Could you please confirm them again.
Thank you.